vllm/model_executor/layers/parameters/lazy_compressed.py

```diff
@@ -56,16 +68,28 @@ def unwrap(e):
             if isinstance(e, LazyCompressedParameter):
                 assert ret_storage_format_cls is None or ret_storage_format_cls == e.storage_format_cls
                 ret_storage_format_cls = e.storage_format_cls
-            return e.uncompressed_data if isinstance(
-                e, LazyCompressedParameter) else e
+
+                if e.is_empty:
+                    e.is_empty = False
+                    e.uncompressed_data = torch.rand(size=e.size(),
```

Comment:
no need to generate random data, please use `torch.empty_like` https://pytorch.org/docs/stable/generated/torch.empty_like.html

Reply:
Fixed

Reply:
empty(..) works, empty_like does not work here (I think empty_like is relaying on .data field that does not exist here)